appchain-registry/src/lib.rs

```diff
@@ -203,13 +215,12 @@ impl AppchainRegistry {
         );
     }
     // Assert that the state of the given appchain is equal to the given AppchainState.
```

Comment:
`Assert that the state of the given appchain is equal to the given AppchainState.`  maybe need to change to `Assert that the state of the given appchain is contained in the given AppchainState set.`

Reply:
OK. Will fix.